src/k100_SteppingAction.cc

```diff
@@ -0,0 +1,48 @@
+#include "k100_SteppingAction.hh"
+#include "k100_EventAction.hh"
+#include "k100_DetectorConstruction.hh"
+
+#include "G4Step.hh"
+#include "G4Event.hh"
+#include "G4RunManager.hh"
+#include "G4LogicalVolume.hh"
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......                                                                                                                                
+
+k100_SteppingAction::k100_SteppingAction(k100_EventAction* eventAction)
+{}
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......                                                                                                                                
+
+k100_SteppingAction::~k100_SteppingAction()
+{}
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......                                                                                                                                
+
```

Comment:
yes, indeed. I suggest lets keep it as it is in case one wants to check states of G4Steps in the passive geometry. I can remove it if you have any objection.